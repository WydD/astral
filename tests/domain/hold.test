<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query t0="auto">
<domain dtype="hold" at="42">
<source attributes="v,PHYID" id="R" sourceAttributes="v,PHYID" type="Relation"/>
</domain>
</query>
<sources>
<source attributes="v,PHYID" file="tests/domain/fix.R" id="R" impl="FileRelationImpl" type="Relation"/>
</sources>
</astral>
