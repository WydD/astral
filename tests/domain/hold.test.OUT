v
t=42
1
t=50
7